main.go

```diff
@@ -22,21 +22,22 @@ import (
 )
 
 var Options struct {
-	AssistedServiceScheme string `envconfig:"ASSISTED_SERVICE_SCHEME"`
-	AssistedServiceHost   string `envconfig:"ASSISTED_SERVICE_HOST"`
-	DataDir               string `envconfig:"DATA_DIR"`
-	HTTPSKeyFile          string `envconfig:"HTTPS_KEY_FILE"`
-	HTTPSCertFile         string `envconfig:"HTTPS_CERT_FILE"`
-	HTTPSCAFile           string `envconfig:"HTTPS_CA_FILE"`
-	ListenPort            string `envconfig:"LISTEN_PORT" default:"8080"`
-	HTTPListenPort        string `envconfig:"HTTP_LISTEN_PORT"`
-	MaxConcurrentRequests int64  `envconfig:"MAX_CONCURRENT_REQUESTS" default:"400"`
-	RHCOSVersions         string `envconfig:"RHCOS_VERSIONS"`
-	OSImages              string `envconfig:"OS_IMAGES"`
-	AllowedDomains        string `envconfig:"ALLOWED_DOMAINS"`
-	InsecureSkipVerify    bool   `envconfig:"INSECURE_SKIP_VERIFY" default:"false"`
-	ImageServiceBaseURL   string `envconfig:"IMAGE_SERVICE_BASE_URL"`
-	LogLevel              string `envconfig:"LOGLEVEL" default:"info"`
+	AssistedServiceScheme        string `envconfig:"ASSISTED_SERVICE_SCHEME"`
+	AssistedServiceHost          string `envconfig:"ASSISTED_SERVICE_HOST"`
+	DataDir                      string `envconfig:"DATA_DIR"`
+	HTTPSKeyFile                 string `envconfig:"HTTPS_KEY_FILE"`
+	HTTPSCertFile                string `envconfig:"HTTPS_CERT_FILE"`
+	HTTPSCAFile                  string `envconfig:"HTTPS_CA_FILE"`
```

Comment:
Let's change this name too:

- Create a new param with a new env var name
- Copy the value from HTTPS_CA_FILE into the new var for backwards compatibility